submodules/externals/gsm

```diff
@@ -1 +1 @@
-Subproject commit b6b37996357ae9d9596b8ee227f33744889d895c
+Subproject commit 57d904f7fdc97e611b4a09ff7753d5d648ff82cd
```

Comment:
Looking at https://github.com/BelledonneCommunications/linphone-iphone, this is b6b37996357ae9d9596b8ee227f33744889d895c, why would I want to use 57d904f7fdc97e611b4a09ff7753d5d648ff82cd?